test/tests/interfaces/pbs_sched_interface_test.py

```diff
@@ -289,3 +289,14 @@ def test_set_default_sched_attrs(self):
                             runas=ROOT_USER)
         self.server.expect(SCHED, {'scheduler_iteration': 500},
                            id='default', max_attempts=10)
+
+    def test_scheduling_iteration(self):
+        """
+        Test scheduler_itration attribute after it is unset. It should go
+        to its default value which is 600. If this happens Server will not
+        kickoff infinite scheduling cycles.
+        """
+        self.server.manager(MGR_CMD_UNSET, SCHED,
+                            ATTR_schedit, id='TestCommonSched')
```

Comment:
You should probably set it to something else first and then unset it.  It's 600 by default, so you don't know if it remained 600 or it was unset and set back to 600.  While you are at it, you might want to check other defaults.  This test seems a bit too specific.

Reply:
Thanks for the suggestion. I have now written elaborated test that covers the above scenario as well.